rewatch/src/format_cmd.rs

```diff
@@ -0,0 +1,147 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
+use anyhow::{bail, Result};
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use std::io::{self, Read, Write};
+use std::fs;
+use rayon::prelude::*;
+use crate::helpers;
+use num_cpus;
+
+pub fn run(
+    stdin_path: Option<String>,
+    all: bool,
+    check: bool,
+    files: Vec<String>,
+    bsc_path: Option<PathBuf>,
+    path: PathBuf,
+) -> Result<()> {
+    let project_root = helpers::get_abs_path(&path);
+    let workspace_root = helpers::get_workspace_root(&project_root);
+
+    let bsc_exe = match bsc_path {
+        Some(path) => helpers::get_abs_path(&path),
+        None => helpers::get_bsc(&project_root, &workspace_root),
+    };
+
+    if check && stdin_path.is_some() {
+        bail!("format -stdin cannot be used with -check flag");
+    }
+
+    if all {
+        if stdin_path.is_some() || !files.is_empty() {
+            bail!("format -all can not be in use with other flags");
+        }
+        format_all(&bsc_exe, check)?;
+    } else if stdin_path.is_some() {
+        format_stdin(&bsc_exe, stdin_path.unwrap())?;
+    } else {
+        format_files(&bsc_exe, files, check)?;
+    }
+
+    Ok(())
+}
+
+
+
+fn format_all(bsc_exe: &Path, check: bool) -> Result<()> {
+    let output = Command::new(std::env::current_exe()?)
```

Comment:
Rewatch doesn't have an `info` command like `rescript` did have. So this call won't work and it also is not future proof as this won't work for monorepos.
@jfrolich would it make sense to call `packages::make()` instead and iterate through all packages?